docs/spark-qualification-tool.md

```diff
@@ -493,6 +493,53 @@ details on limitations on UDFs and unsupported operators.
 11. _Exec Children Node Ids_
 12. _Exec Should Remove_: whether the Op is removed from the migrated plan.
 
+**Parsing Expressions within each Exec**
+
+The Qualification tool look at the expressions in the _Exec_ to extract any duration information.
+Note that some of the Execs will not take any expressions.
+The following table lists the status of expression parsing for each _Exec_.
+
+| Exec                                 | Takes-Expression | Fully-Parsed | Future-Work |
```

Comment:
Ok, I'd argue for a single Full-Support column then, with "x" for those execs that were in "Fully-parsed" and "-" for those that need more work (aka "Future-Work")

Reply:
Agree with @abellina that we can consolidate data in single column for "Fully-parsed" and "Future-Work".

> @nartal1 , is the list of "Execs that don't take any expressions/functions" final? Is this meant to be a list that we are not going to build parsers? in that case I should have a column marking them as not supported?

"Execs that don't take any expressions/functions" is the cause of confusion. I should have rephrased it better. It is basically that Execs may take in expressions but those expressions don't show up in the eventlog.  
For example, If I am joining 2 dataframes of type double:
SortMergeJoin will just have the columnNames in the eventlog.
```
df1.select(ceil(col("num")) as "a").join(df2.select(ceil(col("num")) as "b"), $"a" === $"b")).explain

+- Exchange rangepartitioning(b#70L ASC NULLS FIRST, 200), ENSURE_REQUIREMENTS, [id=#221]
   +- *(5) SortMergeJoin [a#68L], [b#70L], Inner
      :- *(2) Sort [a#68L ASC NULLS FIRST], false, 0
      :  +- Exchange hashpartitioning(a#68L, 200), ENSURE_REQUIREMENTS, [id=#204]
      :     +- *(1) Project [CEIL(num#64) AS a#68L]
      :        +- *(1) Filter isnotnull(CEIL(num#64))
      :           +- *(1) ColumnarToRow

```
Likewise, `CartesianProduct` will be:
```
joinDf1.crossJoin(joinDf2).explain
== Physical Plan ==
CartesianProduct
:- *(1) SerializeFromObject [input[0, int, false] AS value#37]
:  +- Scan[obj#36]
+- *(2) SerializeFromObject [input[0, int, false] AS value#43]
   +- Scan[obj#42]
```

Moved GenerateExec to require parsers section. Since it takes in generator i.e Explode, posExplode which is already supported in Gpu, I had marked this as done. But if the function is inline it will fallback to CPU. So good to have a parser for this Exec.